src/features/amUI/common/Avatar/Avatar.tsx

```diff
@@ -15,26 +15,52 @@ interface AvatarProps {
   icon?: React.ReactNode;
   /** Determines the shape of the avatar */
   profile?: AvatarProfile;
+  /** Organization number */
+  orgnr?: string;
   /** Additional class names for styling */
   className?: string;
+  /** Resource owner logo url */
+  logoUrl?: string;
 }
 
-export const Avatar = ({ icon, name, size = 'md', profile = 'person', className }: AvatarProps) => {
+// className={cn([classes.icon, resource.resourceOwnerLogoUrl && classes.logo])}
```

Comment:
Is this comment saved for later?

Reply:
No. this should be removed! I'll fix it right away!